test/geocel/vg/Vecgeom.test.cc

```diff
@@ -984,8 +1070,8 @@ TEST_F(CmseTest, trace)
             180, 910, 24000, 6000};
         EXPECT_VEC_SOFT_EQ(expected_distances, result.distances);
         static real_type const expected_hw_safety[] = {100, 2.1499999999997,
-            0.52499999999986, 13.023518051922, 6.95, 6.95, 13.023518051922,
-            0.52499999999986, 2.15, 100, 5, 8, 100, 100, 100};
+            10.3027302206745, 13.023518051922, 6.95, 6.95, 13.023518051922,
+            10.3027302206745, 2.15, 100, 5, 8, 100, 100, 100};
```

Comment:
It's great that you've recorded the updated safety values: these are now consistent with the ones returned by Geant4. I suppose we'll have to "edit" these when using the volume-based vecgeom.